modules/social_features/social_swiftmail/social_swiftmail.routing.yml

```diff
@@ -2,6 +2,6 @@ social_swiftmail.settings:
   path: '/admin/config/opensocial/swiftmail'
   defaults:
     _form: 'Drupal\social_swiftmail\Form\SocialSwiftmailSettingsForm'
-    _title: 'Social Swiftmail settings'
+    _title: 'Email notification settings'
```

Comment:
@dmytriikaiun In the menu list, it still says `Email settings`. I think we should also change there to `Email notification settings` to be consistent.